clang/test/dpct/cmake_migation/case_014/input.cmake

```diff
@@ -0,0 +1,29 @@
+set(CPP_VER_98 98)
+set(CPP_VER_11 11)
+set(CPP_VER_14 14)
+set(CPP_VER_17 17)
+set(CPP_VER_20 20)
+set(CPP_VER_23 23)
+set(CPP_VER_26 26)
+
+# In the following cases check for changes after running the
+# pattern-rewritter
+set (CMAKE_CXX_STANDARD CPP_VER_98)
+set (CMAKE_CXX_STANDARD CPP_VER_11)
+set (CMAKE_CXX_STANDARD CPP_VER_14)
+# Using magic number to be diverse
+set (CMAKE_CXX_STANDARD 14)
+
+# In the following cases check for NO changes after running the
+# pattern-rewritter. These test will fail due to limitation in
+# pattern-rewritter.
+# set (CMAKE_CXX_STANDARD CPP_VER_17)
+# set (CMAKE_CXX_STANDARD CPP_VER_20)
+# set (CMAKE_CXX_STANDARD CPP_VER_23)
+# set (CMAKE_CXX_STANDARD CPP_VER_26)
```

Comment:
You can migrate "CMAKE_CXX_STANDARD" to "CPP_VER_17" for line20 to line23 for current migration yaml based rule, and add comments for this rule just for temporary use, and an implicit rule for CMAKE_CXX_STANDARD will be added later on.

Reply:
Done